### PR TITLE
Kubernetes initial setup

### PR DESCRIPTION
**Pull request by mateuszkp96**

### CODE REVIEW

File: k8s/core-service/core-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                secretKeyRef:
+                  name: mongo-core
+                  key: db-password
+            - name: SPRING_DATA_MONGODB_AUTHENTICATION_DATABASE
+              valueFrom:
+                configMapKeyRef:
+                  name: mongo-core-config
+                  key: MONGODB_TDS_CORE_DBNAME
+            - name: SPRING_PROFILES_ACTIVE
+              value: dev
```
Comment (nowpawel96) [inline k8s/core-service/core-deployment.yaml:49]: for now we are deploing with dev profile?

File: k8s/frontend/frontend-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          ports:
+            - containerPort: 80
+          env:
+            - name: GOOGLE_AUTH_PROVIDER
+              valueFrom:
+                secretKeyRef:
+                  name: frontend-conf
+                  key: google-auth-provider
+            - name: LOCAL_API_URL
+              value: https://tourdeshot.teamg.com
```
Comment (nowpawel96) [inline k8s/frontend/frontend-deployment.yaml:30]: https is configured?

File: k8s/core-service/core-deployment.yaml (hunk 1 above)
Comment (mateuszkp96) [reply k8s/core-service/core-deployment.yaml:49]: With dev profile enabled inital data is loaded. In prod mode there will be no profiles - all config will be loaded from kubernetes

File: k8s/frontend/frontend-deployment.yaml (hunk 2 above)
Comment (mateuszkp96) [reply k8s/frontend/frontend-deployment.yaml:30]: For now with self-signed certificate, in future I will use Let's encrypt and cert-manager in kubernetes

**Pull request merged**